packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -1,27 +1,43 @@
 import { AxiosInstance } from 'axios';
-import { ISubscriberPayload } from './subscriber.interface';
+import { ISubscriberPayload, ISubscribers } from './subscriber.interface';
+import { IChannelCredentials } from '@novu/shared';
 
-export class Subscribers {
+export class Subscribers implements ISubscribers {
   private readonly http: AxiosInstance;
 
   constructor(http: AxiosInstance) {
     this.http = http;
   }
 
-  async identify(subscriberId: string, data: ISubscriberPayload) {
+  async identify(subscriberId: string, data: ISubscriberPayload): Promise<any> {
```

Comment:
Let's added the returned type here instead of using any.

Reply:
Sure totally understand you, could you create a discussion for this?

Reply:
Sure i will create one shortly :)

Reply:
https://github.com/novuhq/novu/discussions/592

Reply:
Added https://github.com/novuhq/novu/discussions/592